package.json

```diff
@@ -14,13 +15,15 @@
   },
   "dependencies": {
     "@formio/core": "^1.2.0",
+    "@formio/keys": "github:formio/keys",
```

Comment:
This will make it so that nobody can install it since this is a private repo.

Reply:
@travist how do we better provide the offline license key for the tests in this case?